tests/unittests/algo/test_space.py

```diff
@@ -683,6 +683,26 @@ def test_interval(self):
 
         assert space.interval() == [categories, (-3, 3), (-np.inf, np.inf)]
 
+    def test_capacity(self):
+        """Check whether space capacity is correct"""
+        space = Space()
+        probs = (0.1, 0.2, 0.3, 0.4)
+        categories = ('asdfa', 2, 3, 4)
+        dim = Categorical('yolo', OrderedDict(zip(categories, probs)), shape=2)
+        space.register(dim)
+        dim = Integer('yolo2', 'uniform', -3, 6)
+        space.register(dim)
+
+        assert 24 == space.samplescapacity
```

Comment:
If capacity of categories depends on shape, then this should be (4 * 2) * 6 = 48. I would also write the test as such (with the multiplications instead of directly 48) to ease reading and understanding of the test.

Reply:
Agree, I will make the change